src/main/java/io/github/eliux/mega/MegaUtils.java

```diff
@@ -34,6 +38,10 @@ static LocalDateTime parseFileDate(String dateStr) {
     return LocalDateTime.parse(dateStr, MEGA_FILE_DATE_TIME_FORMATTER);
   }
 
+  static LocalDate parseExpireDate(String dateStr) {
+    return  LocalDateTime.parse(dateStr, MEGA_EXPORT_EXPIRE_DATE_FORMATTER).toLocalDate();
```

Comment:
It is nice that you are using the `DateTimeFormatter` but in a util/generic class this functionality is too specific. This function should be named based on what he does not on what you are using it to solve a particular problem. In this case it would be `parseBasicISODate`, right?